scIB/metrics/kbet.py

```diff
@@ -78,7 +78,7 @@ def kBET(
         embed='X_pca',
         type_=None,
         hvg=False,
-        subsample=0.5,
+        subsample=0.5, #non-functional
```

Comment:
I'd suggest we remove that option completely, i.e also from `kBET_single` and `metrics`